src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,18 @@
-import React from 'react';
+import React from "react";
+import { User } from "../../Types/User";
 
-export const UserInfo: React.FC = () => <>Put the user here</>;
+interface UserInfoProps {
+  user: User | undefined;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
```

Comment:
As a general rule of thumb, let's extract properties from the user: 

({ user: { email, name } })

Could you also do the same for PostInfo?